torch/export/exported_program.py

```diff
@@ -317,16 +324,23 @@ def __call__(self, *args: Any, **kwargs: Any) -> Any:
         additional_inputs = tuple(additional_inputs)
 
         self._check_input_constraints(flat_args_with_path)
-        flat_args = [x[1] for x in flat_args_with_path]
+        flat_args = tuple(x[1] for x in flat_args_with_path)
 
         # NOTE: calling convention is first params, then buffers, then args as user supplied them.
         # See: torch/_functorch/aot_autograd.py#L1034
+        return additional_inputs + flat_args
+
+    def __call__(self, *args: Any, **kwargs: Any) -> Any:
```

Comment:
@angelayi is working on removing this API. we can leave it as-is in the diff.

Reply:
Removing `__call__`? 
What is the replacement?

Reply:
@qihqi we're changing to ep.module()(*args, **kwargs), because in a lot of cases people want a drop in replacement torch.nn.Module, and .module() does that.

Reply:
why not keep `__call__` and it forwards to `self.module()(...)`?

Reply:
that's also okayish to do, but we just feel it might cause some confusion to people, because there're two ways to call the same thing from API perspective.

halfjoking, that's not very pythonic(tm)